aexpect/utils/process.py

```diff
@@ -102,3 +104,32 @@ def process_in_ptree_is_defunct(ppid):
             defunct = True
             break
     return defunct
+
+
+PIPE = subprocess.PIPE
+STDOUT = subprocess.STDOUT
+
+
+if sys.version_info < (3, 2):
+    # Actually we should rewrite `subprocess.Popen` with a complete
```

Comment:
Well actually we can just use `subprocess32` backport and set the `fcntl` there (either as suggested here, or by using `subprocess._set_cloexec(fd, False)`, but as that one is private I'd suggest the `fcntl`.